tembo-cli/src/cli/file_utils.rs

```diff
@@ -53,36 +52,13 @@ impl FileUtils {
         Ok(())
     }
 
-    pub fn download_file(
-        file_path: &str,
-        download_location: &str,
-        recreate: bool,
-    ) -> std::io::Result<()> {
-        let path = Path::new(&download_location);
-        if !recreate && path.exists() {
```

Comment:
@joshuajerin The logic to skip generating the `tembo.toml` file if it exists should stay otherwise init will replace existing tembo.toml file which we don't want.